managed/models/database.go

```diff
@@ -1077,6 +1076,74 @@ var databaseSchema = [][]string{
 		`ALTER TABLE user_flags
 			ADD COLUMN snoozed_pmm_version VARCHAR NOT NULL DEFAULT ''`,
 	},
+	107: {
+		`ALTER TABLE agents ADD COLUMN exporter_options JSONB`,
+		`UPDATE agents SET exporter_options = '{}'::jsonb`,
+		`ALTER TABLE agents ADD COLUMN qan_options JSONB`,
+		`UPDATE agents SET qan_options = '{}'::jsonb`,
+		`ALTER TABLE agents ADD COLUMN aws_options JSONB`,
+		`UPDATE agents SET aws_options = '{}'::jsonb`,
+
+		`ALTER TABLE agents ALTER COLUMN azure_options TYPE JSONB USING to_jsonb(azure_options)`,
+		`UPDATE agents SET azure_options = '{}'::jsonb WHERE azure_options IS NULL`,
+		`ALTER TABLE agents ALTER COLUMN mysql_options TYPE JSONB USING to_jsonb(mysql_options)`,
+		`UPDATE agents SET mysql_options = '{}'::jsonb WHERE mysql_options IS NULL`,
+
+		`ALTER TABLE agents RENAME COLUMN mongo_db_tls_options TO mongo_options`,
+		`UPDATE agents SET mongo_options = '{}'::jsonb WHERE mongo_options IS NULL`,
+
+		`UPDATE agents SET postgresql_options = '{}'::jsonb WHERE postgresql_options IS NULL`,
+
+		`UPDATE agents SET exporter_options = jsonb_set(exporter_options, '{expose_exporter}', to_jsonb(expose_exporter));`,
+		`UPDATE agents SET exporter_options = jsonb_set(exporter_options, '{push_metrics}', to_jsonb(push_metrics));`,
```

Comment:
was wondering if we need a `where push_metrics is not null` here to avoid getting `{"push_metrics": null}`